### PR TITLE
Update applying-labels-to-issues-and-pull-requests.md

### LINKED ISSUE #161: Article should note limit of 100 labels applied to issues and pull requests
### What feature or product is affected?
https://help.github.com/en/github/managing-your-work-on-github/applying-labels-to-issues-and-pull-requests

### What is the new or expected behavior?
A user can apply up to a maximum of 100 labels to issues and pull requests.

### How is the old or inaccurate behavior currently documented?
Currently there is no mention of this in the above help article. I would recommend that we add a note to the article noting this limit.

### Who does this affect?
Any user who uses labels with issues and pull requests.

### What is the impact to users?
If a user applies more than 100 labels to an issue or pull request this can cause problems in the GitHub UI, such as problems closing the issue or pull request. 

### Content to update

In https://help.github.com/en/github/managing-your-work-on-github/applying-labels-to-issues-and-pull-requests:
- Add content about limits to top of page (before procedural)
- Rather than having two tips/notes on this page, we should just turn the current tip into content.

### PR DESCRIPTION
**Pull request by casals**
### Why:

Fixes #161 .

### What's being changed:

Tip section updated to reflect the max labels limit, and the previous text in this field was added to the bottom of the page as a complement to item 4.

### Check off the following:
- [x] All of the tests are passing.
- [x] I have reviewed my changes in staging.
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Review by zeke — APPROVED** [review]
Thanks @casals!

This looks good to me, but I'll defer to @github/docs-content for an authoritative review.

File: content/github/managing-your-work-on-github/applying-labels-to-issues-and-pull-requests.md
Comment (megbird) [inline content/github/managing-your-work-on-github/applying-labels-to-issues-and-pull-requests.md:14]:
```suggestion
**Tip:** You can apply a maximum of 100 labels to issues and pull requests.
```

To keep it consistent and friendly, we like refer to readers in the second person 🥰

**Comment by zeke** [comment]
The staging deployment worked, but for some reason the check is hanging.

Here's the staging URL with the changed page: http://docs-179--patch-1.herokuapp.com/en/free-pro-team@latest/github/managing-your-work-on-github/applying-labels-to-issues-and-pull-requests

I will investigate the staging issue but in the meantime, this can be merged as all the other checks are passing. 👍🏼

File: content/github/managing-your-work-on-github/applying-labels-to-issues-and-pull-requests.md
Comment (megbird) [inline content/github/managing-your-work-on-github/applying-labels-to-issues-and-pull-requests.md:24]: Awesome that you moved this to compliment step 4. I think we can go all the way with it and add it to step 4 😄 what do you think? 

> `4. In the upper-right corner, click **Label**, then start typing the name of an existing label. Click the label's name to associate it with the selected items. You can also apply a label in the Labels drop-down menu within an issue or pull request.`
>   `Issues Milestone assignment drop-down`
>

**Review by megbird — CHANGES_REQUESTED** [review]
@casals Thank you so much for your contribution 🎉  🙌  Just a few small things to note before you merge 😄

File: content/github/managing-your-work-on-github/applying-labels-to-issues-and-pull-requests.md
Comment (casals) [reply content/github/managing-your-work-on-github/applying-labels-to-issues-and-pull-requests.md:24]: Sorry, resolved it by mistake :)
Much better this way - I'll make the necessary changes.

File: content/github/managing-your-work-on-github/applying-labels-to-issues-and-pull-requests.md
Comment (casals) [reply content/github/managing-your-work-on-github/applying-labels-to-issues-and-pull-requests.md:14]: Thanks! Great to know it :)

Comment (megbird) [reply content/github/managing-your-work-on-github/applying-labels-to-issues-and-pull-requests.md:14]:
```suggestion
**Tip:** You can apply a maximum of 100 labels to issues and pull requests.
```

So sorry, I don't think I was clear! You can make these edits in the PR after a review and then I'll get notified that your PR has been changed and I can review it again and approve. 😄  Try merging this suggested change and pushing it up to your branch. ⚡

File: content/github/managing-your-work-on-github/applying-labels-to-issues-and-pull-requests.md
Comment (megbird) [inline content/github/managing-your-work-on-github/applying-labels-to-issues-and-pull-requests.md:21]:
```suggestion
4. In the upper-right corner, click **Label**, then start typing the name of an existing label. Click the label's name to associate it with the selected items. You can also apply a label in the Labels drop-down menu within an issue or pull request.
```

This one too 💖

**Review by megbird — CHANGES_REQUESTED** [review]
Once you push those edits I've suggested, I'll review it again and we can publish this one 👍 🎉

File: content/github/managing-your-work-on-github/applying-labels-to-issues-and-pull-requests.md
Comment (casals) [reply content/github/managing-your-work-on-github/applying-labels-to-issues-and-pull-requests.md]: Sorry, you were very clear - I marked the comments as "resolved" before editing the file, took a call, forgot what I was doing. All requested changes committed now.

**Review by megbird — APPROVED** [review]
Amazing! thank you very much for working on this with me 💛 merging as soon as tests are 🍏 🍏 🍏

**Pull request merged**
